docs/pywbemcli/cmdshelp.rst

```diff
@@ -2255,12 +2253,12 @@ Help text for ``pywbemcli statistics`` (see :ref:`statistics command group`):
       -h, --help  Show this help message.
 
     Commands:
-      reset        Reset the counts in statistics gathered by pywbemcli.
-      server-on    Enable statistics on current server.
-      server-off   Disable statistics on current server.
-      server-show  Display statistics gathered by server.
-      show         Display statistics managed by pywbemcli.
-      status       Show statistics enabled status for client and server.
+      reset        Reset client maintained statistics.
+      server-on    Enable server maintained statistics.
+      server-off   Disable server maintained statistics.
+      server-show  Display server maintained statistics.
+      show         Display client maintained statistics.
+      status       Show enablement status of client and server maintained statistics.
```

Comment:
Can we change the word enablement to "enabled".

Reply:
DONE